ehitchhiking-ui/src/app/shared/services/api.services/black-list-api.service.ts

```diff
@@ -0,0 +1,47 @@
+import {User} from '@shared/models/user';
+import {ApiService} from '@shared/services/api.services/api.service';
+import {Car} from '@shared/models/car';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+import {map} from 'rxjs/operators';
+import {Injectable} from '@angular/core';
+
+@Injectable()
+export class BlackListApiService {
+	constructor(private apiService: ApiService) {}
+
+	mapBlackListPassengers(data): User[] {
```

Comment:
mapBlackListPassengers and mapBlacklistDrivers are equal. Pls leave one of them and rename to mapBlackListUser

Reply:
+ add type

Reply:
done